test/modules/TestResultCoordinator/config/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "OptimizeForPerformance": "true",
+  "StoragePath": "",
+  "ResultSources": "loadGen1.send,messages;loadGen2.send,messages;relayer1.receive,messages;relayer2.receive,messages;relayer;relayer1.send,messages;relayer2.send,messages"
```

Comment:
Can we make this an array? Or an object?

Reply:
If you decide to go with a stringified json array and deserialize in the module, this might be hard to integrate into the pipeline considering the backslash escape chain you have to work around (which behaves differently for linux and windows). I've ran into this issue before which caused problems. If you want to represent objects that aren't stringified, it might be a good idea to set this value in the module twin's desired properties rather than in the environment vars.

Reply:
You should be able to use objects and arrays.. take a look at this example - 
https://github.com/Azure/iotedge/blob/master/edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/DependencyManager.cs#L111

Reply:
Varun shared that you can use colon (:) in envrionment variable to represent a json value.  For example, key1: { key1.1:value1.1, key1.2: value1.2}; environment variables should be "key1:key1.1"="value1.1" and "key1:key1.2"="value1.2".

Reply:
After discussion with Varun, we will hard-code for now and enhance later.

Reply:
So we've decided to go with environment variables as objects. But we are going to combine it with the configuration for the Report types. So we are going to do this at a later date.

I've hard coded a sample list that encapsulates the first version of tests we will make, and I've added a TODO to change it when we decide exactly how we will have those env vars.